L1Trigger/TrackFindingTracklet/interface/DTC.h

```diff
@@ -0,0 +1,80 @@
+// This class holds a list of stubs that are in a given layer and DCT region
+#ifndef DTC_H
+#define DTC_H
+
+#include "L1TStub.h"
+#include "Stub.h"
+#include "DTCLink.h"
+
+using namespace std;
```

Comment:
Please don't `using namespace std` in a header file.

Reply:
fixed by migrating to h/cc files